### ISSUE TITLE
fcgi meson error

### ISSUE THREAD

**Issue by benwaffle** [valum-framework/valum#212]
```
FAILED: src/vsgi/servers/vsgi-fastcgi@sha/vsgi-fastcgi.c src/vsgi/servers/vsgi-fastcgi.h src/vsgi/servers/vsgi-fastcgi.vapi 
valac -C --debug --enable-experimental --enable-deprecated /home/ben/.cache/pacaur/valum/src/valum-0.3.13/src/vsgi/servers/fcgi.vapi -lfcgi --pkg libsoup-2.4 --pkg gio-unix-2.0 --pkg gio-2.0 --pkg gobject-2.0 --target-glib 2.40 --pkg glib-2.0 -d src/vsgi/servers/vsgi-fastcgi@sha --library=vsgi-fastcgi -H src/vsgi/servers/vsgi-fastcgi.h --use-header --vapi ../vsgi-fastcgi.vapi --define=GIO_2_44 --define=SOUP_2_48 ../src/vsgi/servers/vsgi-fastcgi.vala src/vsgi/vsgi-0.3.vapi
Unknown option -lfcgi
```

**Comment by arteymix** [comment]
This will be addressed in `0.41.1` and I already filed a bug https://github.com/mesonbuild/meson/issues/1939

**Comment by arteymix** [comment]
In the meantime, use `pip3 install meson==0.40.1`!
